src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Recipes/ElasticsearchIndexStep.cs

```diff
@@ -26,20 +26,33 @@ ElasticsearchIndexManager elasticIndexManager
 
     protected override async Task HandleAsync(RecipeExecutionContext context)
     {
-        if (context.Step["Indices"] is not JsonArray jsonArray)
-        {
-            return;
-        }
+        // Elasticserach uses the term "Indices" for the plural of index, but the common spelling in US English is
+        // "Indexes". So both are supported to avoid unnecessary spelling problems.
+        var indexes = context.Step["Indexes"] as JsonArray ??
+                      context.Step["Indices"] as JsonArray ??
+                      [];
```

Comment:
see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/index-indexes-indices

Reply:
Do we support both step names?

Reply:
The step name is `ElasticIndexSettings`, the `Indices`/`Indexes` is just a custom property inside the settings object (see example [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/5e29e6727929989d9f25ab5e1c979bf6e6f14697/src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.Elasticsearch.recipe.json#L28)) so it can be whatever we want. I'd like to add support for the more common American spelling to avoid unnecessary spelling problems, but the `Indices` should be kept for backwards compatibility.